sirepo/auth_db.py

```diff
@@ -340,10 +336,11 @@ def get_status(cls, uid, role):
                 return s.status
 
         @classmethod
-        def set_status(cls, uid, role, status):
+        def set_status(cls, uid, role, status, moderator_uid=None):
             with sirepo.util.THREAD_LOCK:
```

Comment:
Assert status is one of the valid values. Normally the db would do this if it were an enum, but we don't want to get into that.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c